### DISCUSSION TITLE
Display poster image on Embedded player (Play.html) instead of text message "Stream will start playing automatically when it is live"

### DISCUSSION THREAD

**Discussion by yashtandon113** [ant-media/Ant-Media-Server#4877] [FAQs] [answered]
How can we have the embedded player display a poster image instead of a text slate when the stream is not playing?

**Comment by Mohit-3196** [comment, marked as answer]
You can add an image or logo to ant media server embedded web player (play.html) as

Here are the steps:

1. Add logo.png file to the images directory.

images directory is under `usr/local/antmedia/webapps/WebRTCAppEE/images`

2. Edit play.html file as suggested in the [PR](https://github.com/ant-media/StreamApp/pull/217/files/9c71ab2936cbd12eade95a27557f28ab1849974b)

The text message "Stream will start playing automatically when it is live" can be edited or removed completely by editing play.html page as well.

Thank you

**Reply by yashtandon113** [reply]
In v2.5.3 and earlier, you had to modify the text or put the logo on three separate portions of the code, as recommended in the PR on the above response, depending on the **playOrder** type. Now, in v2.6.0 and higher, you may quickly alter the text or insert a logo by making a little update to the play.html file.

**1. To change the text "Your live stream will play automatically as soon as it's available. <br />Get ready to be part of the action!"**

   Edit `play.html` under `/usr/local/antmedia/webapps/App-Name/` folder and change the text as required.
    
**2. To put the logo in place of text or along with text.**

<ol>
  <li>Open your terminal and navigate to the directory where Ant Media Server is installed:<br>
  <code>cd /usr/local/antmedia/{APP_NAME}</code></li>
  
  <li>Use your favorite text editor to open the <code>play.html</code> file:<br>
  <code>sudo nano play.html</code></li>
  
  <li>In the <code>play.html</code> file, search for the "logo" section in the text. It should look like this:<br>
  <pre><code>  &lt;div&gt;
    &lt;img id="logo" src="images/logo.png" /&gt;
  &lt;/div&gt;</code></pre></li>
  
  <li>Remove the commented-out parts (<code>&lt;!--</code> and <code>--&gt;</code>) <br>

  <li>This will display a logo on the player side </li>
</ol>
<img src="https://github.com/ant-media/Ant-Media-Server/assets/47350008/4a7b942c-03d3-43c4-b181-63d03daa47a2" width=400 >

<p>To change the default logo image:</p>

<ol start="6">
  <li>Navigate to the directory where the logo image file is located:<br>
  <code>cd /usr/local/antmedia/{APP_NAME}/images/</code></li>
  
  <li>In this directory, you will find a file called <code>logo.png</code>. Replace this file with your logo image and also change the name in html file if the name is other then logo.png</li>
</ol>
